internal/pkg/template/templates/workloads/partials/cf/service-base-properties.yml

```diff
@@ -21,6 +21,17 @@ DeploymentConfiguration:
     AlarmNames: {{ .DeploymentConfiguration.RollbackAlarms }}
     Enable: true
     Rollback: true
+  {{- else if or .DeploymentConfiguration.CPUUtilization .DeploymentConfiguration.MemoryUtilization }}
+  Alarms:
+    AlarmNames:
+      {{- if .DeploymentConfiguration.CPUUtilization }}
+      - !Sub ${AWS::StackName}-CPUAlarm
```

Comment:
Will CFN support something like `!GetAtt CPURollbackAlarm.Name`? With `!Sub ${AWS::StackName}-CPUAlarm`, we will have to keep in mind that this needs to match the actual alarm name written in `rollback-alarms` - which is probably not ideal in the long run.

Reply:
It appears the only available attribute for that resource is `ARN`, unfortunately.

Reply:
Yeah also it'd introduce a circular dependency between the `CPUUtilizationAlarm` and `Service` : (